unit_test.sh

```diff
@@ -9,6 +9,13 @@ set -o pipefail
 echo "Running linter..."
 golangci-lint run
 
+timestamp=`date +%Y-%m-%d_%H-%M-%S`
+mkdir -p test_results/${timestamp}
+mkdir -p test_results/latest
+
```

Comment:
Since you are creating `timestamp` and the two directories here, can you remove these lines:

https://github.com/CMSgov/bcda-app/pull/75/files#diff-12c47e9f6d43cdddab176851f23f31dbR31
https://github.com/CMSgov/bcda-app/pull/75/files#diff-12c47e9f6d43cdddab176851f23f31dbR33
https://github.com/CMSgov/bcda-app/pull/75/files#diff-12c47e9f6d43cdddab176851f23f31dbR34